ssh-find-agent.sh

```diff
@@ -140,7 +140,9 @@ find_all_agent_sockets() {
 	_LIVE_AGENT_LIST=$(echo $_LIVE_AGENT_LIST | tr ' ' '\n' | sort -n -t: -k 2 -k 1 | uniq)
 	_LIVE_AGENT_SOCK_LIST=()
 	_debug_print "SORTED: $_LIVE_AGENT_LIST"
-  _FINGERPRINTS=$(fingerprints ~/.ssh/authorized_keys)
+	if [ -e ~/.ssh/authorized_keys ] ; then
+  		_FINGERPRINTS=$(fingerprints ~/.ssh/authorized_keys)
```

Comment:
Maybe this variable is orphaned?

Reply:
I think so, I can't find another reference to it.